### PR TITLE
test/mpi: xfail gather_big for UCX

### PR DESCRIPTION
**Pull request by pavanbalaji**

### CODE REVIEW

**Comment by pavanbalaji** [comment]
Doing a sanity check:

test:jenkins/ch4/ucx

**Comment by pavanbalaji** [comment]
@hzhou Can you review this when you get a chance?  This PR doesn't change the existing xfails to point to corresponding GitHub issues, like we had discussed on #3770, but I pointed this new xfail to the corresponding GitHub issue.

File: test/mpi/maint/jenkins/xfail.conf [hunk 1]
```diff
@@ -40,6 +40,7 @@ ch4 * * * * sed -i "s+\(^idup_comm_gen .*\)+\1 xfail=ticket0+g" test/mpi/threads
 * * * ch4:ucx * sed -i "s+\(^win_large_shm .*\)+\1 xfail=ticket0+g" test/mpi/rma/testlist
 * * * ch4:ucx * sed -i "s+\(^strided_putget_indexed_shared .*\)+\1 xfail=ticket0+g" test/mpi/rma/testlist
 * * * ch4:ucx * sed -i "s+\(^bcastlength .*\)+\1 xfail=ticket0+g" test/mpi/errors/coll/testlist
+* * * ch4:ucx * sed -i "s+\(^gather_big .*\)+\1 xfail=issue3770+g" test/mpi/coll/testlist
 # am-only failures
```
Comment (hzhou) [inline test/mpi/maint/jenkins/xfail.conf]: We have now 5 xfail entries of `gather_big` and start to lose track on which is still on. Should we group them together?

Comment (hzhou) [reply test/mpi/maint/jenkins/xfail.conf]: and we might as well add `issue3770` tag to all of them.

Comment (pavanbalaji) [reply test/mpi/maint/jenkins/xfail.conf]: The rest are not covered by that issue, so that'd be incorrect.

Comment (pavanbalaji) [reply test/mpi/maint/jenkins/xfail.conf]: Also, the rest are not `gather_big`.

**Comment by pavanbalaji** [comment]
test:jenkins/ch4/ucx

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8

**Comment by pavanbalaji** [comment]
@hzhou Is the custom test orthogonal to this PR?  If yes, can you signoff on this?  Thanks.

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8
testlist: coll/gather_big 8 env=MPIR_CVAR_GATHER_INTRA_ALGORITHM=nb env=MPIR_CVAR_GATHER_DEVICE_COLLECTIVE=0 env=MPIR_CVAR_IGATHER_DEVICE_COLLECTIVE=0 env=MPIR_CVAR_IGATHER_INTRA_ALGORITHM=gentran_tree env=MPIR_CVAR_IGATHER_TREE_KVAL=8

**Comment by hzhou** [comment]
test:mpich/custom
testlist: coll/gather_big 8 timeLimit=3600
testlist: coll/gather_big 8 timeLimit=3600
    : env=MPIR_CVAR_GATHER_INTRA_ALGORITHM=nb
    : env=MPIR_CVAR_GATHER_DEVICE_COLLECTIVE=0 
    : env=MPIR_CVAR_IGATHER_DEVICE_COLLECTIVE=0 
    : env=MPIR_CVAR_IGATHER_INTRA_ALGORITHM=gentran_tree 
    : env=MPIR_CVAR_IGATHER_TREE_KVAL=8

**Pull request merged**